framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -529,8 +529,7 @@ protected void addInformationFromPreconditions(
                                 false);
                 info.insertValue(expr, annotation);
             } catch (FlowExpressionParseException e) {
-                // report errors here
-                analysis.checker.report(e.getResult(), methodTree);
```

Comment:
Summary of discussion: without flow, one gets a parse error only at the method declaration. With flow, one gets an additional error at method invocations. Post-conditions are only used for flow refinement, so it would be unnecessary overhead to also check them at invocations.